tests/utilities.py

```diff
@@ -0,0 +1,79 @@
+# coding=utf-8
+"""Helper module for gui test suite."""
+
+import sys
+from qgis.utils import iface
+
+
+QGIS_APP = None  # Static variable used to hold hand to running QGIS app
+CANVAS = None
+PARENT = None
+IFACE = None
+
+
+def get_qgis_app():
```

Comment:
Can we remove this function? This is legacy code I think.

In Cadasta : 
https://github.com/Cadasta/cadasta-qgis-plugin/blob/develop/cadasta/gui/tools/test/test_cadasta_project_creation.py#L24

In InaSAFE : 
https://github.com/inasafe/inasafe/blob/develop/safe/test/utilities.py#L68

I would prefer having a function, like in InaSAFE.